scripts/constants.js

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
great, I'd love to know a good way of structuring things, so every file can depend on constants hosted somewhere